contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -431,10 +425,27 @@ Go to the root of the source code, and execute the following build command:
 
 ::
 
+    PATH="$RISCV_TOOLCHAIN_PATH/bin:$PATH"
     scons arch=rv64 use_llvm=yes linker=mold lto=none target=editor \
```

Comment:
Won't work as-is, you either need to put it on one line or add a `\` to the end.

```suggestion
    PATH="$RISCV_TOOLCHAIN_PATH/bin:$PATH" scons arch=rv64 use_llvm=yes linker=mold lto=none target=editor \
```

Reply:
Ok, I thought about it, but wasn't shure. Done!